game-core/src/main/java/games/strategy/triplea/ai/pro/logging/ProLogWindow.java

```diff
@@ -384,8 +386,11 @@ void addMessage(final Level level, final String message) {
         currentLogTextArea = aiOutputLogArea;
       }
       currentLogTextArea.append(message + "\r\n");
-    } catch (final NullPointerException ex) { // This is bad, but we don't want TripleA crashing because of this...
```

Comment:
I vote let it crash due to an NPE.  That's probably the only way the root cause will get fixed. :smile:

Reply:
In general I agree, was tempted to do so here; but really needed to not alter functionality too much (many many bad memories of small changes adding hours).

More serious, it's the same IMO as logging quietly, our quiet logging is not 'debug/info', it's suppressed error logging. But, a cleanup here would be good to see, we do have a lot of inappropriate catch clauses and needless catch exceptions (not to mention control-flow-by-exception-handling)